gensim/models/fasttext_inner.pyx

```diff
@@ -633,7 +643,7 @@ def train_batch_sg(model, sentences, alpha, _work, _l1):
         Learning rate.
     _work : np.ndarray
         Private working memory for each worker.
-    _l1 : np.ndarray
+    _neu1 : np.ndarray
```

Comment:
Why this change?

Reply:
99%-similar methods `train_batch_cbow` & `train_batch_sg` were merged, & the naming in the `_cbow` version won out.